github/orgs_properties.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CustomProperty represents an organization custom property object.
+type CustomProperty struct {
+	PropertyName *string `json:"property_name,omitempty"`
```

Comment:
This field is required, so let's please make it mandatory.

```suggestion
	PropertyName string `json:"property_name"`
```

Reply:
This was a conscious change due to https://docs.github.com/en/rest/orgs/custom-properties?apiVersion=2022-11-28#create-or-update-a-custom-property-for-an-organization. This API relies on `property_name` to be specified in the path instead of the field to be in the custom property object.

Reply:
Hmm, just to clarify. Would it be fine to update it to send the unused field `property_name` as part of the body?

Reply:
Ah, I see what you are talking about.

How about just add a comment to this field... something like:
```go
// Although PropertyName is required for most endpoints, this struct is shared with the PUT endpoint
// where it is sent in the path, and is therefore a pointer with "omitempty".
```

Reply:
Thank you @gmlewis. I'll update it with something along those lines.